File: C++/Problem_001.cpp
#include "stdafx.h"
#include <iostream>
#include <numeric>
#include <algorithm>
#include <vector>
/*Author: Christian Seely*/

int problemOne()
{
  std::vector<int> v(1000);
  std::iota(v.begin(), v.end(), 0);
  v.erase(std::remove_if(v.begin(), v.end(), [](const int& i) {return !(i % 3 == 0 || i % 5 == 0);}), v.end());
  return std::accumulate(v.begin(), v.end(), 0);
}

int main()
{
  std::cout << problemOne() << std::endl;
  return 0;
}

File: Java/Problem_001.java
import java.util.stream.IntStream;

/**
 * 
 * @author Christian Seely
 *
 */

public class Problem_001 {

    public static int problemOne(){
        return IntStream.range(0,1000).filter(i -> i % 3 == 0 || i % 5 == 0).sum();
    }

    public static void main(String[] args) { System.out.println(problemOne()); }
}